mlflow/runs.py

```diff
@@ -0,0 +1,74 @@
+"""
+CLI for runs
+"""
+import click
+import datetime
+import json
+from mlflow.entities import ViewType
+from mlflow.tracking import _get_store
+from tabulate import tabulate
+
+RUN_ID = click.argument("run_id", type=click.STRING)
```

Comment:
There are philosophical differences between using arguments (`mlflow runs restore a1234`) and options (`mlflow runs restore --run-id a1234`) in CLIs.

In the context of MLflow, I have argued in the past that in most cases, options allow for a more self-documenting and flexible API, at the cost of verbosity. Self-documenting is pretty obvious -- as a reader of `mlflow runs restore --run-id a1234`, I can figure out that `a1234` is a run-id, otherwise it could be a git hash or something.

Flexibility is a bit less obvious, but I think it's relevant in this case. Suppose that tomorrow, we want people to be able to restore runs either by run id or by run name -- if we are using arguments, we may have to either infer the new type or add a "--type=name" sort of indicator. If we're using options, we can simply add a "--run-name" and require that a user provide either a "--run-id" or "--run-name".  This is definitely a more intuitive API!

Anyway, all that is to say I would recommend we switch to `--run-id` as an option.

Reply:
Good point! I'm defiantly agree with you.
I would change It to --run-id.